arch/x86_64/src/common/up_assert.c

```diff
@@ -250,7 +250,7 @@ void up_assert(const char *filename, int lineno)
 
   board_autoled_on(LED_ASSERTION);
 
-#if CONFIG_TASK_NAME_SIZE > 0
+#if CONFIG_TASK_NAME_SIZE > 0 && defined(CONFIG_DEBUG_ALERT)
```

Comment:
why need check CONFIG_DEBUG_ALERT additionally?

Reply:
to match the above variable declaration.